components/builder-api/habitat/default.toml

```diff
@@ -1,3 +1,6 @@
+[srv]
+port = 9636
```

Comment:
Is this actually supposed to configure the port that builder-api is running on or is it only a convenience mechanism for `pkg_expose`? (configuring the port would be my impression.)